builds/misc/images-release.yaml

```diff
@@ -1,354 +1,476 @@
-  - job: manifest
+  - stage: PublishManifests
 ################################################################################
-    displayName: Publish Manifest Images
+    displayName: Publish Manifests
     pool:
-      vmImage: 'ubuntu-18.04'
-    dependsOn:
-      - linux_dotnet_projects
-    steps:
-    # Both docker logins needed for if we need to test this job. In this case images should go to edgebuilds.
-    - task: Docker@2
-      displayName: Docker login edgebuilds
-      inputs:
-        command: login
-        containerRegistry: iotedge-edgebuilds-acr
-    - task: Docker@2
-      displayName: Docker login edgerelease
-      inputs:
-        command: login
-        containerRegistry: iotedge-release-acr
-    - script: scripts/linux/buildManifest.sh -r '$(registry.address)' -v '$(version)' -t '$(System.DefaultWorkingDirectory)/edge-modules/iotedge-diagnostics-dotnet/docker/manifest.yaml.template' -n '$(namespace)' --tags '$(tags)'
-      displayName: 'Publish azureiotedge-diagnostics Manifest'
-    - script: scripts/linux/buildManifest.sh -r '$(registry.address)' -v '$(version)' -t '$(System.DefaultWorkingDirectory)/edge-agent/docker/manifest.yaml.template' -n '$(namespace)' --tags '$(tags)'
-      displayName: 'Publish Edge Agent Manifest'
-    - script: scripts/linux/buildManifest.sh -r '$(registry.address)' -v '$(version)' -t '$(System.DefaultWorkingDirectory)/edge-hub/docker/manifest.yaml.template' -n '$(namespace)' --tags '$(tags)'
-      displayName: 'Publish Edge Hub Manifest'
-    - script: scripts/linux/buildManifest.sh -r '$(registry.address)' -v '$(version)' -t '$(System.DefaultWorkingDirectory)/edge-modules/SimulatedTemperatureSensor/docker/manifest.yaml.template' -n '$(namespace)' --tags '$(tags)'
-      displayName: 'Publish Temperature Sensor Manifest'
\ No newline at end of file
+      name: $(pool.linux.name)
+      demands:
+        - ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker
+    dependsOn: BuildImages
+    jobs:
+    - job: PublishManifest
+      displayName: Publish Manifest
+      strategy:
+        matrix:
+          EdgeAgent:
+            manifestFilePath: '$(System.DefaultWorkingDirectory)/edge-agent/docker/manifest.yaml.template'
+          EdgeHub:
+            manifestFilePath: '$(System.DefaultWorkingDirectory)/edge-hub/docker/manifest.yaml.template'
+          SimulatedTemperatureSensor:
+            manifestFilePath: '$(System.DefaultWorkingDirectory)/edge-modules/SimulatedTemperatureSensor/docker/manifest.yaml.template'
+          Diagnostics:
+            manifestFilePath: '$(System.DefaultWorkingDirectory)/edge-modules/iotedge-diagnostics-dotnet/docker/manifest.yaml.template'
+      steps:
+      # Both docker logins needed for if we need to test this job. In this case images should go to edgebuilds.
+      - task: Docker@2
+        displayName: Docker login edgebuilds
+        inputs:
+          command: login
+          containerRegistry: iotedge-edgebuilds-acr
```

Comment:
We don't need the `edgebuilds` login. It only needs to be here for testing purposes.

Reply:
Yes, this was included to support testing. It was included before my changes too (see the manifest job that's marked as removed in this PR diff). I've also included the comment above explaining why both docker logins are needed.